site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
+
+On the other hand, with `AT_LEAST_ONCE` delivery, clients indicate that they want to get a response if the message was successfully delivered to the downstream peer and, in the case of failure, be informed that the message cannot be delivered. To this end, the AMQP adapter distinguishes between two types of failures:
+
+* A failure caused by a client-side error (e.g invalid message content type, missing device-id in address, adapter disabled for tenant, etc).
+* A failure caused by a server-side error (e.g sender queue full, no credit available yet to send the message, etc).
+
+For a client-side error, the adapter responds with the *REJECTED* disposition and provides a reason why the message was rejected together with the error condition value. If a bad request is the reason why the message cannot be processed, then the adapter returns a `hono:bad-request` as the error condition value. If the message cannot be processed because the adapter was disabled for the tenant, then the adapter returns the `amqp:unauthorized-access` error condition value. For all other client-side errors, the adapter returns the `amqp:precondition-failed` as error condition value. 
+
+In the case of a server-side error, the adapter responds with a *RELEASED* disposition, which indicates to the client that the message was OK but it cannot be delivered due to a failure beyond the control of the client. In this case, the client should attempt to redeliver the message again.
+
+<!--more-->
+
+## Device Authentication
+
+By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during connection establishment. This is the preferred way for devices to publish data via protocol adapters. The AMQP adapter can be configured to *disallow* authentication for devices by setting the value of `HONO_AMQP_AUTHENTICATION_REQUIRED` to `false`. In this guide, we will give examples for publishing telemetry and events for *authenticated* and *unauthenticated* clients. We will see in the examples later that the main difference between publishing messages for authenticated and unauthenticated devices is in the format of the AMQP 1.0 **address**. For unauthenticated devices, the address should include the tenant and device identifier.
+
+### SASL PLAIN Authentication
+
+The AMQP adapter currently supports authenticating clients using a *username* and *password*. This means that clients needs to provide a *username* and a *password* when connecting to the AMQP adapter. The *username* must match the pattern [*auth-id@tenant*], e.g. `sensor1@DEFAULT_TENANT`. The adapter verifies the credentials provided by the client against the credentials that the [Credentials Service] ({{< relref "#credentials-service-connection-configuration" >}}) has on record for the device. If the credentials match, then authentication is successful and the client device can proceed to publish messages to Hono.
```

Comment:
I think you should link to the RFC that defines SASL PLAIN for reference here ...